### PR TITLE
Fix that fields were editable when they shouldn't be

### LINKED ISSUE #1212: Bug: Fields are editable when user is not assigned

### PR DESCRIPTION
**Pull request by MatanYadaev**

### CODE REVIEW

File: apps/backoffice-v2/src/pages/Entity/components/EditableDetails/EditableDetails.tsx [hunk 1]
```diff
@@ -284,6 +284,7 @@ export const EditableDetails: FunctionComponent<IEditableDetails> = ({
                         {isInput && !isSelect && (
                           <FormControl>
                             <Input
+                              {...field}
```
Comment (MatanYadaev) [inline apps/backoffice-v2/src/pages/Entity/components/EditableDetails/EditableDetails.tsx:287]: `field` had a `disabled` property that was set to `undefined`. This was reproduced only when the backoffice was installed as a standalone project (not part of a monorepo)

File: services/workflows-service/.env.example [hunk 2]
```diff
@@ -16,7 +16,13 @@ API_KEY=secret
 NODE_ENV=local
 SENTRY_DSN=
 WEBHOOK_SECRET=webhook_secret
-IS_DEMO=
+WEBHOOK_URL=
```
Comment (tomer-shvadron) [inline services/workflows-service/.env.example:19]: Is this necessary to be in the code-base?

**Review by tomer-shvadron — APPROVED** [review]

File: apps/backoffice-v2/src/pages/Entity/components/EditableDetails/EditableDetails.tsx (hunk 1 above)
Comment (Omri-Levy) [reply apps/backoffice-v2/src/pages/Entity/components/EditableDetails/EditableDetails.tsx:287]: Great work! 👑

**Review by Omri-Levy — APPROVED** [review]

File: services/workflows-service/.env.example (hunk 2 above)
Comment (MatanYadaev) [reply services/workflows-service/.env.example:19]: Currently, yes

**Pull request merged**